<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>端口扫描</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>漏洞扫描</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>开启任务</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>获取任务</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>上传数据</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>生成chat</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="configTab">
       <attribute name="title">
        <string>任务配置</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>检测名单</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>13.126.139.188</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>端口范围</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="text">
           <string>1-20000</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>NMAP 命令</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>-Pn -O -T 3 -sV --version-intensity=9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-Pn -O -T 3 -sV --version-intensity=5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-Pn -T 2 -sV --version-intensity=4</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>HOST</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="text">
           <string>http://192.168.133.132:18002</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>USERNAME</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="text">
           <string>admin</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>PASSWORD</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="text">
           <string>048c60971e056793518850b8f1a5bd56bab153a12f303d1296d2b4844330f19c3ce98c8972f908722f9e80aa371d9e4a3667d64f9f57f6d3483b63a647dd3270af0f6dc59fb294fd65bcf2dbe03ca6d96e726fc931eb65f280c7e7a42025f422180c2c1b1253a82873</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>任务列表</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>taskname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>domain</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>state</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>actionstime</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>actionetime</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>progress</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>资产信息</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>domain</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ip</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>os_info</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>port</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>service</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>protocol</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>version</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>漏洞数据</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_2">
          <column>
           <property name="text">
            <string>ip</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>lat</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>lon</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>cve_id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>cve_name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>cve_cvss</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>cve_cvss3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>cve_summary</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>日志信息</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>图表</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
